controllers/chat/group/groupAddMembers.js

```diff
@@ -45,14 +45,15 @@ const groupAddMembers = async (req, res) => {
 
   const channel = await client.queryChannels({
     id: channel_id,
-    type: CHANNEL_TYPE_STRING.GROUP,
-    members: {$in: [userId]}
+    type: CHANNEL_TYPE_STRING.GROUP /* ,
+    members: {$in: [userId]} */
   });
-
   if (channel?.length < 1) {
     return ErrorResponse.e404(res, 'Group not found');
   }
 
+  let all_members = channel[0].data.better_channel_member.map((member) => member.user.id);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The commented out code on lines 48 and 49 should be removed to maintain clean and readable code. Also, the `all_members` variable is not defined in the old hunk but it's used later in the new hunk. Make sure that this change doesn't introduce any bugs or inconsistencies.
```diff
-    type: CHANNEL_TYPE_STRING.GROUP /* ,
-    members: {$in: [userId]} */
+    type: CHANNEL_TYPE_STRING.GROUP,
+    members: {$in: [userId]}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->